### PR TITLE
Add release note for Gradle init changes

### PR DESCRIPTION
**Pull request by alllex**
See https://github.com/gradle/gradle/pull/27270

### CODE REVIEW

File: platforms/documentation/docs/src/docs/release/notes.md [hunk 1]
```diff
@@ -89,6 +89,10 @@ Execution failed for task ':dependencies'.
 > Get more help at https://help.gradle.org.
 ```
 
+### Gradle init
+
+When using `gradle init` to generate projects with Kotlin DSL scripts they are now using [simple property assignment](/8.4/release-notes.html#assign-stable) syntax with `=` operator.
```
Comment (lkasso) [inline platforms/documentation/docs/src/docs/release/notes.md:94]:
```suggestion
Projects generated using `gradle init` with Kotlin DSL scripts now use [simple property assignment](/8.4/release-notes.html#assign-stable) syntax with the `=` operator.
```

**Review by lkasso — APPROVED** [review]
Small cleanup.

**Comment by alllex** [comment]
@lkasso, I added some more notes after https://github.com/gradle/gradle/pull/27264 got merged

**Comment by alllex** [comment]
@bot-gradle merge

**Pull request merged**